sbom/main.ts

```diff
@@ -42,7 +42,7 @@ function getHomepage(pkg: Package) {
 /**
  *  Generates the document header for the produced SBOM.
  */
-async function generateDocumentHeader(docNamespace: string) {
+function generateDocumentHeader(docNamespace: string) {
```

Comment:
Then we can
* Remove the `await` in https://github.com/phylum-dev/community-extensions/blob/e6aac63248406571a9311c2367fee5eaa6d9c459/sbom/main.ts#L93
* Remove the `async` in https://github.com/phylum-dev/community-extensions/blob/e6aac63248406571a9311c2367fee5eaa6d9c459/sbom/main.ts#L91
* Remove the `await` in https://github.com/phylum-dev/community-extensions/blob/e6aac63248406571a9311c2367fee5eaa6d9c459/sbom/main.ts#L131